lesson24_cp1/project_users/views.py

```diff
@@ -0,0 +1,72 @@
+import hashlib
+from django.db.utils import IntegrityError
+from django.shortcuts import render
+from django.views import View
+from project_users.models import CustomUser
+
+
+# Create your views here.
+
+class HomePageView(View):
+    TITLE = "Home Page"
+
+    def get(self, request, *args, **kwargs):
+        count = CustomUser.objects.count()
+        data = CustomUser.objects.filter(pk__gte=count - 10)
```

Comment:
🤔 Lets imagine I created users with ids 1, 255, 3, 4, 5, 6, 7, 8, 9, 10, 11 and.. 155